core/chains/evm/txmgr/orm.go

```diff
@@ -95,6 +87,34 @@ type orm struct {
 	ctxCancel context.CancelFunc
 }
 
+func ToQOpt(opt any) (pg.QOpt, error) {
+	qopt, ok := opt.(pg.QOpt)
+	if !ok {
+		return func(*pg.Q) {}, ErrInvalidQOpt
+	}
+	return qopt, nil
+}
+
+func ToQOpts(opts []any) ([]pg.QOpt, error) {
```

Comment:
These methods are for converting between  []pg.QOpt and []any

Go can't convert slices natively from `[]pg.QOpt` (or any slice for that matter) to `[]any` (see https://go.dev/doc/faq#convert_slice_of_interface)

However, Go can convert between `pg.QOpt` (or any type for that matter) and `any`